lib/active_admin/csv_builder.rb

```diff
@@ -44,7 +44,7 @@ def column(name, options = {}, &block)
     def build(controller, csv)
       @collection  = controller.send :find_collection, except: :pagination
       columns      = exec_columns controller.view_context
-      options      = ActiveAdmin.application.csv_options.merge self.options
+      options      = @options
```

Comment:
We don't need this line at all. `options` (used below) already means reading the instance variable `@options`, right?

Reply:
You are right. I totally missed the attr_reader for options